servatrice/src/servatrice.cpp

```diff
@@ -258,6 +258,12 @@ bool Servatrice::initServer()
 		qDebug() << "Forgot password challenge on: " << getEnableForgotPasswordChallenge();
 	}
 
+	qDebug() << "Auditing enabled: " << getEnableAudit();
```

Comment:
Yeah I caught that and mentioned it in the gitter chat.  I'll role through and detab.